pegasus/helpers/section_api_helpers.rb

```diff
@@ -6,6 +6,15 @@ def self.admin?(user_id)
   end
 end
 
+class DashboardUser
+  def self.to_self_view(user_row)
```

Comment:
It felt like the name was getting less and less descriptive... `to_self_hash` on a hash sounds like a no-op.  Also, this one's a static method.  So knowing that `slice` exists, maybe I should ditch this method altogether and just call slice in the route definition.